examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -79,10 +81,11 @@ echo "    Slurm output dir: $ROOT_DIR"
 cp buildenv/submit.daint.slurm .
 sed s/\<NAME\>/standalone/g submit.daint.slurm -i
 sed s/\<NTASKS\>/$ranks/g submit.daint.slurm -i
-sed s/\<NTASKSPERNODE\>/$ranks/g submit.daint.slurm -i
-sed s/\<CPUSPERTASK\>/1/g submit.daint.slurm -i
-sed s/#SBATCH\ --output=\<OUTFILE\>//g submit.daint.slurm -i
+sed s/\<NTASKSPERNODE\>/1/g submit.daint.slurm -i
+sed s/\<CPUSPERTASK\>/nthreads/g submit.daint.slurm -i
```

Comment:
`$nthreads`?

Reply:
Possibly move `-i` in front for these sed commands, as is customary?

Reply:
is it? I am the worst basher - open to any suggestions here